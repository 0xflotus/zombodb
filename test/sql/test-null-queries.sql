select dsl.geo_shape('field', null, 'INTERSECTS');
select dsl.geo_polygon('field', null);
select dsl.geo_bounding_box('field', null);