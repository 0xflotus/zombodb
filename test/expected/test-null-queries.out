select dsl.geo_shape('field', null, 'INTERSECTS');
 geo_shape 
-----------
 
(1 row)

select dsl.geo_polygon('field', null);
ERROR:  null points are not allowed
select dsl.geo_bounding_box('field', null);
 geo_bounding_box 
------------------
 
(1 row)

